.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: php
+
+sudo: true
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
```

Comment:
Missing `nightly`.

Reply:
I know but build failed with nightly. Even if there is 'allowed failures'